dataset/companies/OpenAI.yaml

```diff
@@ -0,0 +1,13 @@
+
+- Name: "OpenAI"
+  Description: "AI Research makers of ChatGPT"
+  Website: "https://openai.com/"
+  alternatives: ["Hugging Face Open Source Moadels - https://huggingface.co/"]
+- Name: "OpenAI - ChatGPT"
+  Description: "AI ChatBot"
+  Website: "https://chat.openai.com/"
+  alternatives: ["Hugging Face Chat - https://huggingface.co/"]
```

Comment:
Alternatives should be a list of comapnies like this 

```
Alternatives: [
{
"Name": "Hugging Face Chat",
"Website": "https://huggingface.co/"
}
]
```

I will update the documentation and introduce data validation

Reply:
If we're using YAML, then perhaps...
```yaml
  Alternatives:
    - Name: "Hugging Face Chat",
      Website: "https://huggingface.co/"
```

Reply:
@hosamaly We have introduced them for the company and the Alternatives, Please check the latest change on the main branch

Reply:
and, it should be yaml 😄